cron.go

```diff
@@ -14,7 +12,7 @@ import (
 
 	"github.com/iancoleman/strcase"
 	"github.com/pkg/errors"
-	"go.etcd.io/etcd/v3/clientv3"
+	client "go.etcd.io/etcd/client/v3"
```

Comment:
Can you update all files so that this import is named `etcdclient`? I find `client` a bit too blurry..

```suggestion
	etcdclient "go.etcd.io/etcd/client/v3"
```